### PR TITLE
update templates for test-core

### PR DESCRIPTION
**Pull request by KPostOffice**

### CODE REVIEW

File: openshift/cronJob-template.yaml [hunk 1]
```diff
@@ -73,6 +73,12 @@ objects:
                 - image: "${IMAGE_STREAM_REGISTRY}/${IMAGE_STREAM_NAMESPACE}/package-update-job:${IMAGE_STREAM_TAG}"
                   name: package-update
                   env:
+                    - name: KAFKA_BOOTSTRAP_SERVERS
+                      value: "stage-kafka-bootstrap-dh-stage-message-bus.cloud.paas.stage.psi.redhat.com:443"
```
Comment (goern) [inline openshift/cronJob-template.yaml]: @harshad16 please open another PR to get this from configmap thoth and be created by ansible.

File: openshift/cronJob-template.yaml [hunk 2]
```diff
@@ -73,6 +73,12 @@ objects:
...
+                    - name: KAFKA_CAFILE
+                      value: "/mnt/secrets/kafka_ca.crt"
+                    - name: KAFKA_TOPIC_RETENTION_TIME_SECONDS
+                      value: 60*60*24
```
Comment (fridex) [inline openshift/cronJob-template.yaml]: Just curious - when/how is this evaluated?

File: openshift/cronJob-template.yaml (hunk 1 above)
Comment (harshad16) [reply openshift/cronJob-template.yaml]: cool will do :+1:

File: openshift/cronJob-template.yaml (hunk 2 above)
Comment (KPostOffice) [reply openshift/cronJob-template.yaml]: This actually currently isn't getting evaluated, I have the value hardcoded in messages/base_message.py

**Comment by KPostOffice** [comment]
recheck

**Pull request merged**